src-ui/components/multi/MappingPane.cpp

```diff
@@ -146,6 +146,94 @@ struct MappingZoneHeader : juce::Component
     }
 };
 
+struct Zoomable : public juce::Component
```

Comment:
This is a super useful class. Do we want to put it in its own header and factor it to use in other places? Fine if not to do that now of course but food for thought.

Reply:
Agree